popup/popup.js

```diff
@@ -0,0 +1,183 @@
+const projects = [
+  {
+    title: 'TONIC 1',
+    prof: ['Canopy', 'back end Dev', '2015'],
+    src: 'images/SnapshootPortfolio.png',
+    description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
+    technologies: ['html', 'css', 'javascript', 'Github', 'Ruby', 'Bootstrap'],
+    seeLiveLink: '#',
+    seeSourceLink: '#',
+  },
+  {
```

Comment:
- [ ] You should stick to the Figma design :+1:  , my fellow code reviewer did not suggest that , however it's an important part of the Project . Kindly improve that to make it beautiful :baby: 

[Screencast from 02-03-2023 01_53_47 PM.webm](https://user-images.githubusercontent.com/80458199/216598014-07b40f43-1cdf-4477-8365-2e03f2fb96e7.webm)